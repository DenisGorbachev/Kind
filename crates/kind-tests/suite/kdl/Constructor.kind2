#kdl_name = Kek
#keep
type Bool {

    #kdl_name = Ata
    #keep
    true

    #kdl_name = Bee
    #keep
    false
}