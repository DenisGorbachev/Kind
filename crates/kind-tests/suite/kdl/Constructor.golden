ctr {Ata}
ctr {Kek}
ctr {Bee}
